syntax="proto2";

import "aap_protobuf/shared/MessageStatus.proto";

package aap_protobuf.service.sensorsource.message;

message SensorStartResponseMessage
{
    required shared.MessageStatus status = 1;
}
